tangelo/problem_decomposition/incremental/mifno_helper.py

```diff
@@ -295,7 +252,7 @@ def compute_fermionoperator(self, molecule, frag_id):
 
         return new_molecule._get_fermionic_hamiltonian(mo_coeff)
 
-    def mi_summation(self, user_provided_energies=None, force_negative_epsilon=False):
+    def mi_summation(self, user_provided_energies=None):
```

Comment:
is the negative epsilon thing now not done in Tangelo anymore ? Maybe it happens on QEMIST Cloud instead ?

Reply:
It was mainly for debugging purposes (for small fragments, where MI might be variational), but it shouldn't be enabled to get the same energy as in QEMIST Cloud. However, MI is not variational in a general sense, and I think keeping this may be confusing.